src/config/dynamic-theme-fixes.config

```diff
@@ -12471,6 +12471,19 @@ INVERT
 
 ================================
 
+vod.tvp.pl
+
+INVERT
+img[alt="serwisy tvp"]
+
+CSS
```

Comment:
You can remove this, stumbled upon a bug and should be fixed within the code. Anyway, thanks for fixing it and accidentally stumbling upon a bug 😄

Reply:
you meant remove CSS section only and save INVERT image?

Reply:
Yep!